bindings/python/native/src/client/types.rs

```diff
@@ -81,6 +84,13 @@ pub struct MilestoneDto {
     pub message_id: String,
 }
 
+#[derive(Debug, Clone, DeriveFromPyObject, DeriveIntoPyObject)]
+pub struct MilestoneUtxoChanges {
+    pub index: u32,
+    pub created_outputs: Vec<String>,
```

Comment:
instead of `Vec<String>`, IMO we should have `Vec<UTXOInput>` in order to match `get_output()`

Reply:
Yeah makes sense, will change that